package.json

```diff
@@ -14,10 +14,10 @@
   },
   "license": "MIT",
   "dependencies": {
-    "jsdom": "^9.8.3"
+    "jsdom": "^12.2.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.16.0",
-    "babel-preset-es2015": "^6.16.0"
+    "babel-cli": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1"
```

Comment:
It's recommended to use `@babel/present-env`

Reply:
Same as above